CRM/Usermenu/Hooks/CoreResourceList/HomeMenu.php

```diff
@@ -43,8 +45,9 @@ protected function shouldRun() {
     $homeMenu = $this->getHomeMenu();
     $isHeaderRegion = $this->region === 'html-header';
     $isHomeMenuActive = $homeMenu && $homeMenu['is_active'] == 1;
```

Comment:
Maybe also change the `getHomeMenu` function to also return only active menu by default same as is done for the `usermenu` for consistencty?